File: sample/src/main/java/com/lany192/box/sample/data/binder/ImageBinder.java
package com.lany192.box.network.data.binder;

import androidx.annotation.NonNull;

import com.github.lany192.arch.items.ItemBinder;
import com.github.lany192.utils.ImageUtils;
import com.lany192.box.sample.databinding.ItemImageBinding;

public class ImageBinder extends ItemBinder<String, ItemImageBinding> {

    @Override
    public void convert(@NonNull ItemImageBinding binding, String url, int position) {
        ImageUtils.show(binding.image, url);
    }
}

File: sample/src/main/java/com/lany192/box/sample/data/binder/ImageBinder.kt
package com.lany192.box.sample.data.binder

import com.github.lany192.arch.extension.load
import com.github.lany192.arch.items.ItemBinder
import com.lany192.box.sample.databinding.ItemImageBinding

class ImageBinder : ItemBinder<String, ItemImageBinding>() {

    override fun convert(binding: ItemImageBinding, item: String, position: Int) {
        binding.image.load(item)
    }
}